Makefile

```diff
@@ -1,6 +1,9 @@
 
 # Image URL to use all building/pushing image targets
 IMG ?= controller:latest
+SIDECAR_IMG ?= quay.io/csiaddons/k8s-sidecar
```

Comment:
the image name looks more generic here and we should not use `sidecar` in the name. can be named something else? am also not getting any solid name as of now.

Reply:
`k8s-sidecar` is fine with me, it is part of the csi-addons repository already too.